Add references to terminology whitepaper (#109)
confidential-computing/governance: terminology/common-terminology.md

While the aforementioned terms help describe how a TEE may be packaged, this does not imply that a specific packaging will meet a meaningful set deployment requirements.  As a result, it is important to define a minimal set of functional prerequisites which a deployable instance of the package must incorporate.  Where these functional prerequisites are not met in deployment, the aforementioned terms themselves should not be used.

> Mutually Assured Secure Channel – Confidential information will go to and/or from a TEE.  To accomplish this in a way which inhibits man-in-the-middle attacks / hijacking, a technology which allows a Mutually Assured Secure Channel must be used in both provision time and runtime.  One viable technology for this function includes TLS with Negotiated Finite Field Diffie-Hellman (RFC 7919) [^4] key exchange.  

Reviewer: Looks good to me

> Mutually Assured Secure Channel – Confidential information will go to and/or from a TEE.  To accomplish this in a way which inhibits man-in-the-middle attacks / hijacking, a technology which allows a Mutually Assured Secure Channel must be used in both provision time and runtime.  One viable technology for this function includes TLS with Negotiated Finite Field Diffie-Hellman (RFC 7919) [^4] key exchange.  

Author: ```suggestion
Mutually Authenticated and Encrypted Channel – Confidential information will go to and/or from a TEE.  To accomplish this in a way which inhibits man-in-the-middle attacks / hijacking, a technology which allows a mutually authenticated and encrypted channel must be used in both provision time and runtime, such as Transport Layer Security (TLS) Protocol Version 1.3 (RFC 8446) [^4].

```

Steve to propose something short here.

## References

> [^1] Confidential Computing Consortium, "A Technical Analysis of Confidential Computing", January 2021, <https://confidentialcomputing.io/whitepaper-02-latest/>.

Reviewer 2: formatting
```suggestion
[^1]: Confidential Computing Consortium, "A Technical Analysis of Confidential Computing", January 2021, <https://confidentialcomputing.io/whitepaper-02-latest/>.
```

> [^2] Open Container Initiative, <https://opencontainers.org/>.

Reviewer 2: ```suggestion
[^2]: Open Container Initiative, <https://opencontainers.org/>.
```

> [^3] Nieles, M., Dempsey, K., and V. Pilliteri, "An Introduction to Information Security", NIST 800-12 Rev.1, June 2017, <https://csrc.nist.rip/publications/detail/sp/800-12/rev-1/final>.

Reviewer 2: ```suggestion
[^3]: Nieles, M., Dempsey, K., and V. Pilliteri, "An Introduction to Information Security", NIST 800-12 Rev.1, June 2017, <https://csrc.nist.rip/publications/detail/sp/800-12/rev-1/final>.
```

> [^4]: Gillmor, D., "Negotiated Finite Field Diffie-Hellman Ephemeral Parameters for Transport Layer Security (TLS)", RFC 7919, DOI 10.17487/RFC7919, August 2016, <https://www.rfc-editor.org/info/rfc7919>.

Author: ```suggestion
[^4]: Rescorla, E., "The Transport Layer Security (TLS) Protocol Version 1.3", RFC 8446, DOI 10.17487/RFC8446, August 2018, <https://www.rfc-editor.org/info/rfc8446>.
```
